packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -31,16 +40,16 @@ export interface PanelChromeProps {
   padding?: PanelPadding;
   title?: string;
   titleItems?: PanelChromeInfoState[];
-  menu?: React.ReactElement;
-  /** dragClass, hoverHeader, loadingState, and states not yet implemented */
+  menu?: ReactElement;
+  /** dragClass, hoverHeader not yet implemented */
   // dragClass?: string;
   hoverHeader?: boolean;
-  // loadingState?: LoadingState;
-  // states?: ReactNode[];
-  /** @deprecated in favor of prop states
+  loadingState?: LoadingState;
+  status?: Status;
```

Comment:
what is this for?

Reply:
ah, see now. ignore comment :)

Reply:
Error of the data, absolutely positioned in the top middle part

Reply:
ah sorry, second comment didn't show up